test/e2e/copy.test.js

```diff
@@ -55,4 +56,23 @@ describe('Handy pg copy test', () => {
     .then(() => pg.copyTo(writeStream, 'films'))
     .then(() => expect(fs.readFileSync(sourcePath)).to.eql(fs.readFileSync(destinationPath)));
   });
+
+  describe('Using an external pool', () => {
+    it('copies from read stream to table', async () => {
+      const handyPostgres = createPostgres({ configPath: 'withSql' });
```

Comment:
this may be confusing - if you're providing your own pool why would you need to pass the config in?

Reply:
I agree. I'm trying to fix it.

Reply:
I have tried removing the config here, I agree on you. The issue is that because of that, line 69 fails, as it has no way to know what 'select-all' means.

Reply:
Though you are passing your own pool, you need to pass config for the route to the formatted queries files and the migrations @feliun

Reply:
cool - let's see how that looks in the rest of the tests